pkg/utils/etcdutil/etcdutil.go

```diff
@@ -592,7 +594,6 @@ func (lw *LoopWatcher) load(ctx context.Context) (nextRevision int64, err error)
 				log.Error("run post event failed in watch loop", zap.String("name", lw.name),
 					zap.String("key", lw.key), zap.Error(err))
 			}
-			log.Info("load finished in watch loop", zap.String("name", lw.name), zap.String("key", lw.key))
```

Comment:
Why remove it

Reply:
There is a duplicated log, you can search it.